### DISCUSSION TITLE
Metrics to monitor total repositories/onboarding and errors

### DISCUSSION THREAD

**Discussion by chandapukiran** [renovatebot/renovate#26223] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### What is your question?

Hi, 

I am currently testing the renovate metrics using otel exporter and need some help to understanding the following. I am testing it with single repository.  Each time i run renovate using github action the count for operation onboarding and repositories in increasing but as per my understanding the value should stay 1 because I only have one repository i.e onboarded. 

```
calls_total{label1="value1",operation="onboarding",service_name="renovate",span_kind="SPAN_KIND_INTERNAL",status_code="STATUS_CODE_UNSET"} 3
calls_total{label1="value1",operation="repository",service_name="renovate",span_kind="SPAN_KIND_INTERNAL",status_code="STATUS_CODE_UNSET"} 3
```

Also, is it possible to get metrics for PRs Open, Merged and closed

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by HonkingGoose** [comment]
Here's some inspiration:

- #23105 (Make sure you expand the full discussion!)
- [Renovate docs, Swissquote user story](https://docs.renovatebot.com/user-stories/swissquote/)
- #23961
